test/functionalTest/cases/1068_metadata_value_as_compound/metadata_value_as_compound_array.test.DISABLED

```diff
@@ -59,13 +61,21 @@ echo
 
 echo "02. GET /v2/entities, to see the entity"
 echo "======================================="
+payload='{
```

Comment:
Yeah :-)
Not sure how this got in there, probably a cut 'n paste error ... 
The commented option --noPayloadCheck will be removed as well.